functions/package.json

```diff
@@ -10,9 +12,29 @@
     "node": "14"
   },
   "bin": {
-    "import": "./src/bin/import.js"
+    "app-search-firestore-extension": "./lib/bin/import.js"
```

Comment:
I couldn't get the npx command to run without changing this to `app-search-firestore-extension`, I'm, not 100% clear why.

This was pointing to `src` and not `lib` originally which was invalid.